### ISSUE TITLE
Reactjs issue 2

### ISSUE THREAD

**Issue by hoanghiep** [nikhilkalige/docblockr#196]
```
/**
 * [createClass description]
 * @param  {[type]} render [description]
 * @return {[type]}        [description]
 */
var QuitJob = React.createClass({
  render : function(){
    return(<div></div>);
  }
});
```

 * @param  {[type]} render [description]

"render" is not a param?

**Comment by MoritzKn** [comment]
I can reproduce this. As pointed out in [#195](https://github.com/nikhilkalige/docblockr/issues/195#issuecomment-237216069) I think this happens due to the same regex problem.

**Comment by MoritzKn** [comment]
#199 partly fixed this, but now it will generate a comment like this:
```
/**
 * [QuitJob description]
 * @type {RegExp}
 */
```
The type `RegExp` will be recognized because [`JsParser.prototype.guess_type_from_value`](https://github.com/nikhilkalige/docblockr/blob/master/lib/languages/javascript.js#L152) just 'RegExp' for everything that contains a slash.

**Comment by MoritzKn** [comment]
The RegExp problem was fixed with #207.

**Comment by hoanghiep** [comment]
Thank you so much, <3 docblockr
